e2e/suites/provider/cases/conjur/provider.go

```diff
@@ -100,98 +104,84 @@ func makeStore(name, ns string, c *addon.Conjur) *esv1.SecretStore {
 	}
 }
 
-func (s *conjurProvider) CreateApiKeyStore(c *addon.Conjur, ns string) {
+func (s *conjurProvider) CreateApiKeyStore() {
 	By("creating a conjur secret")
 	conjurCreds := &v1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      ns,
-			Namespace: ns,
+			Name:      s.framework.Namespace.Name,
+			Namespace: s.framework.Namespace.Name,
 		},
 		Data: map[string][]byte{
-			"apikey":   []byte(c.AdminApiKey),
+			"apikey":   []byte(s.addon.AdminApiKey),
 			"username": []byte("admin"),
 		},
 	}
-	err := s.framework.CRClient.Create(context.Background(), conjurCreds)
+	err := s.framework.CRClient.Create(GinkgoT().Context(), conjurCreds)
 	Expect(err).ToNot(HaveOccurred())
 
 	By("creating an secret store for conjur")
-	secretStore := makeStore(ns, ns, c)
+	secretStore := makeStore(s.framework.Namespace.Name, s.framework.Namespace.Name, s.addon)
```

Comment:
In couple of places you have conjur either hardcoded or setting things like this:
```go
l.ConjurURL = fmt.Sprintf("https://conjur-%s-conjur-oss.%s.svc.cluster.local", l.Namespace, l.Namespace)
```

And now we don't really know which one is being used. :D

Reply:
My intention (with my limited capacity) is just to restructure the way we run the vault/conjur tests. I honestly don't care too much about the tests or the naming scheme of a provider we don't officially maintain. Let me see if i can "easily" clean that up. Renaming is usually a big mess :face_with_spiral_eyes:

Reply:
good 'ol `conjur-conjur-conjur-oss` service :D

Reply:
Perfect. :D